src/init.cpp

```diff
@@ -1327,7 +1330,16 @@ bool AppInitMain(NodeContext& node, interfaces::BlockAndHeaderTipInfo* tip_info)
 
     // ELEMENTS:
     policyAsset = CAsset(uint256S(gArgs.GetArg("-feeasset", chainparams.GetConsensus().pegged_asset.GetHex())));
```

Comment:
Maybe put this in else of the switch because this has sense only with single asset fee

Reply:
Makes sense. Changed in: https://github.com/SequentiaSEQ/SEQ-Core-Elements/pull/6/commits/69bc2ebca466da066622e233d77e8376a78a7961